llmfoundry/command_utils/data_prep/convert_dataset_hf.py

```diff
@@ -451,6 +451,7 @@ def convert_dataset_hf_from_args(
         ValueError: If the output directory already contains the requested splits
         ValueError: If `concat_tokens` is set but `tokenizer` is not
     """
+    os.environ['WORLD_SIZE'] = '1'
```

Comment:
are these changes supposed to be included?

Reply:
Actually going to close this pr, during testing seems like every version I tested worked (strangely), so perhaps old bug was actually transient, and was solved not by the version change but simply restarting the cluster. Will revisit if it comes up again.

Reply:
Original motivation is we don't know what works and what gets deprecated